converter/shieldhit/beam.py

```diff
@@ -126,7 +130,8 @@ class BeamConfig:
 
     beam_dat_template: str = """
 RNDSEED      	89736501     ! Random seed
-JPART0       	2            ! Incident particle type
+JPART0       	{particle}            ! Incident particle type
```

Comment:
Do we have somewhere a mapping between particle names and particle codes ?
We have it here in UI:
https://github.com/yaptide/ui/blob/6a78d703af8d49e4c181f1bdf8395db082453169/src/types/Particle.ts#L38

It would be nice to have here in a comment a particle name, not only code.

There are two ways to implement this:
 - when user defines particle source in UI, the particle name is also sent in the JSON to backend, not only the particle code
 -  add here the dictionary with mapping id <-> names and use it to get useful comment

I would suggest second option as the easiest